packages/sdk/client/src/echo/space-list.ts

```diff
@@ -142,7 +142,7 @@ export class SpaceList extends MulticastObservable<Space[]> implements Echo {
           this._spaceCreated.emit(spaceProxy.key);
 
           if (this._defaultSpaceId && spaceProxy.id === this._defaultSpaceId) {
-            this._onDefaultSpaceAvailable();
+            this._defaultSpaceAvailable.next(true);
```

Comment:
this goes against the jsdoc in echo-protocol, as the observable now is not completed